### DISCUSSION TITLE
Disable build cache

### DISCUSSION THREAD

**Discussion by lodmfjord** [vercel/next.js#37618] [Ideas]
### Describe the feature you'd like to request

Disable build cache

The cache directory can get quite huge for some apps while the output stays small. This is unfortunate when using something like TurboRepo with remote caching to cache the .next folder. 

### Describe the solution you'd like

Being able to disable build cache. 

### Describe alternatives you've considered

For my use case, I guess it would be okay to have the cache directory outside of .next? 

I also could skip this from TurboRepo, which I am not sure if is possible now, but hopefully:
https://github.com/vercel/turborepo/issues/1366

**Comment by anatolzak** [comment]
You can disable build cache by setting the `VERCEL_FORCE_NO_BUILD_CACHE` environment variable to `1` in your project.

**Reply by ddadaal** [reply]
This env doesn't seem to work In 13.2.1. Is this env still available?  

We are distributing our next.js project as docker images, and build cache is of no use for us but takes over 100M size.

![image](https://user-images.githubusercontent.com/8363856/223116091-33377e33-1366-4561-a0e4-9096c69a1293.png)

**Comment by haifn** [comment]
same problem, 
setting VERCEL_FORCE_NO_BUILD_CACHE environment variable not work, still creates cache during build phase
dir: .next/cache 
How to properly disable caching during the build phase.

Next.js Version: 12.0.7

**Reply by haifn** [reply]
@anatolzak
